src/tests/dispatch-payload.test.ts

```diff
@@ -170,7 +181,7 @@ Deno.test("DispatchPayload with unhandled events", async (t) => {
         body: payload.body,
         env: payload.context.variables,
         team_id: payload.context.team_id,
-        enterprise_id: "",
+        enterprise_id: payload.body.enterprise_id,
```

Comment:
This change was needed due to how `extractBaseHandlerArgsFromPayload` now works: it looks for `enterprise_id` in two different locations.

One potential TODO is to do a round of testing to see what the different payloads look like for the different events, in both an enterprise and non-enterprise context, possibly document them in this repo somewhere (maybe in `types.ts`?) and make sure the property extractions are still accurate.